src/mmw/apps/water_balance/templates/home_page/index.html

```diff
@@ -208,88 +216,88 @@ <h1 class="infil">0.0 cm</h1>
                         </a>
-                        <label>Tall Grass Prairie</label>
+                        <label>Pasture/Hay</label>
                     </li>
 
-                    <li id="thumb-short_grass_prairie"> <!-- Short-Grass Thumb -->
-                        <a href="#land-short_grass_prairie" aria-controls="filter-tab" role="tab" data-toggle="tab">
-                            <img src="{% static 'images/water_balance/thumb_shortGrass.png' %}" alt="Short Grass Prairie" data-container="body" data-toggle="popover" data-placement="left" data-title="Short Grass Prairie" data-content="Areas dominated by shrubs less than 20 centimeters tall with shrub canopy typically greater than 20% of total vegetation. This type is often co-associated with grasses, sedges, herbs, and non-vascular vegetation.">
+                    <li id="thumb-row_crop">
+                        <a href="#land-row_crop" aria-controls="filter-tab" role="tab" data-toggle="tab">
+                            <img src="{% static 'images/water_balance/thumb_rowCrops.png' %}" alt="Cultivated Crops" data-container="body" data-toggle="popover" data-placement="left" data-title="Cultivated Crops" data-content="Areas used for the production of annual crops, such as corn, soybeans, vegetables, tobacco, and cotton, and also perennial woody crops such as orchards and vineyards. Crop vegetation accounts for greater than 20% of total vegetation. This class also includes all land being actively tilled.">
                         </a>
-                        <label>Short Grass Prairie</label>
+                        <label>Crops</label>
                     </li>
 
-                    <li id="thumb-desert"> <!-- Desert Thumb -->
-                        <a href="#land-desert" aria-controls="filter-tab" role="tab" data-toggle="tab">
-                            <img src="{% static 'images/water_balance/thumb_desert.png' %}" alt="Desert" data-container="body" data-toggle="popover" data-placement="left" data-title="Desert" data-content="Areas of bedrock, desert pavement, scarps, talus, slides, volcanic material, glacial debris, sand dunes, strip mines, gravel pits and other accumulations of earthen material. Generally, vegetation accounts for less than 15% of total cover.">
+                    <li id="thumb-woody_wetland">
+                        <a href="#land-wetland" aria-controls="filter-tab" role="tab" data-toggle="tab">
+                            <img src="" style="background-color: gray;" alt="Wetlands" data-container="body" data-toggle="popover" data-placement="left" data-title="Dwarf Scrub" data-content="(Including wooded wetland, herbaceous wetland, marsh, bog). Areas where forest or shrubland vegetation accounts for greater than 20% of vegetative cover and the soil or substrate is periodically saturated with or covered with water.">
```

Comment:
Data title should not be "Dwarf Scrub". Description should read "woody wetland" instead of "wooded wetland" to match NLCD.